plugins/hydra_ray_launcher/integration_test_tools/create_integration_test_ami.py

```diff
@@ -29,7 +29,7 @@ def _run_command(command: str) -> str:
     return output
 
 
-@hydra.main(config_name="create_integration_test_ami_config")
+@hydra.main(config_path=None, config_name="create_integration_test_ami_config")
```

Comment:
This change is unrelated to python3.9 support, right?

Reply:
Yes, this is just to fix the warning.